src/Entity/User.php

```diff
@@ -138,7 +148,7 @@ public function getRoles(): array
     {
         $roles = $this->roles;
         // guarantee every user at least has ROLE_USER
-        $roles[] = 'ROLE_USER';
+        $roles[] = 'ROLE_UNVERIFIED';
```

Comment:
You should add `const ROLE_UNVERIFIED = 'ROLE_UNVERIFIED';` on line 27 of this file, and then just call it here: `$roles[] = self::ROLE_UNVERIFIED;`

Reply:
Done